src/test/java/org/jabref/gui/autocompleter/FieldValueSuggestionProviderTest.java

```diff
@@ -56,6 +60,23 @@ void completeAfterAddingEntryWithoutFieldReturnsNothing() {
         assertEquals(Collections.emptyList(), result);
     }
 
+    @Test
+    void completeOnIgnoredFieldReturnsNothing() {
+        AutoCompletePreferences autoCompletePreferences = mock(AutoCompletePreferences.class);
+        JournalAbbreviationRepository journalAbbreviationRepository = mock(JournalAbbreviationRepository.class);
+        when(autoCompletePreferences.getCompleteFields()).thenReturn(Set.of(StandardField.AUTHOR));
+        SuggestionProviders suggestionProviders = new SuggestionProviders(database, journalAbbreviationRepository, autoCompletePreferences);
+
+        SuggestionProvider<String> autoCompleter = (SuggestionProvider<String>) suggestionProviders.getForField(StandardField.TITLE);
+
+        BibEntry entry = new BibEntry();
+        entry.setField(StandardField.TITLE, "testValue");
```

Comment:
`new BibEntry().withField()`

Reply:
I have to admit I was not aware of existence of this method, this would certainly be an improvement. The thing is that all the tests are written in the manner I did it. It would probably be a good idea to change them all at once for the sake of consistency. This however sounds like something for another PR.